src/Models/Services/Utils.ts

```diff
@@ -648,6 +648,35 @@ export function sortAlphabetically<T>(
     };
 }
 
+/**
+ * Sort function to order items from ascending or descending order, for boolean, numbers and strings. Case insensitive sort
+ * NOTE: only works when property is one layer down
+ * @param propertyName name of the property to sort on
+ * @returns Sort function to pass to `.sort()`
+ */
+export function ascendOrDescendSort<T>(
```

Comment:
can we please not create yet another sort function? We already have a ton of them scattered in the app. This should functionally be the same as the sortAlphabetcially function. if you don't like that name, maybe we just rename it to this name?

Reply:
yeah sortAlphabetically is not the best name anymore since it does more than strings. And i decided to make it a separate function because i was trying to take the less 'intrusive' route.

Reply:
Yeah. I get that. Are you cool with just doing the rename as the last commit now that we are done reviewing and we can just delete the existing function? Otherwise it's just more cleanup later. :)

Reply:
yup definitely. i'll do that now